lib/trace.js

```diff
@@ -461,14 +461,16 @@ Trace.prototype.getLoadRecord = function(canonical, excludeURLs, parentStack) {
             if (typeof sourceMap == 'string')
               sourceMap = load.metadata.sourceMap = JSON.parse(sourceMap);
 
-            var originalName = load.name.split('!')[0];
-            
-            // force set the filename of the original file
-            sourceMap.file = originalName + '!transpiled';
-
-            // force set the sources list if only one source
-            if (!sourceMap.sources || sourceMap.sources.length <= 1)
-              sourceMap.sources = [originalName];
+            if (load.name.indexOf('!') != -1) {
```

Comment:
https://github.com/systemjs/systemjs/blob/master/lib/plugins.js#L202 is obviously already filtered to this plugin case...